libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -1618,10 +1700,12 @@ async def atransform(
 
 
 def patch_config(
-    config: RunnableConfig, callback_manager: BaseCallbackManager
+    config: RunnableConfig, callbacks: BaseCallbackManager, **kwargs: Any
 ) -> RunnableConfig:
     config = config.copy()
-    config["callbacks"] = callback_manager
+    config["callbacks"] = callbacks
+    # Correct typing requires Unpack, which isn't supported in all Python versions
```

Comment:
I think typing_extensions can be installed to use Unpack

Reply:
No, it requires a Mypy flag